app/controllers/api/vms_controller.rb

```diff
@@ -47,7 +47,7 @@ def shelve_offload_resource(type, id = nil, _data = nil)
     def edit_resource(type, id, data)
       edit_resource_with_genealogy(type, id, data)
     rescue => err
-      raise BadRequestError, "Cannot edit VM - #{err}"
+      raise BadRequestError, _("Cannot edit VM - #{err}")
```

Comment:
This error can't be interpolated this way for i18n purposes and instead should use placeholders (see the i18n guide here: https://github.com/ManageIQ/guides/blob/master/i18n.md#string-interpolation)

Reply:
Also below wherever there is interpolation